### ISSUE TITLE
Up/Down Pagination arrows not working on Dialogue page

### ISSUE THREAD

**Issue by RussellSPG** [samtecspg/articulate#791]
### Issues
For some reason, the up/down arrows, and next button are not functional to change pages on dialogue. Before, I did use the up-down arrows. After inputting new sayings without actions this happened I believe.  

Refreshing did fix the issue and cannot re produce at this team. Will be annoying quirk to deal with none the less.

No errors on console:

Step 8.1 on script.

Terminal:
```
s, loss=0.112, acc=0.933]2019-06-27 17:56:02+0000 [-] "172.18.0.6" - - [27/Jun/2019:17:56:02 +0000] "POST /train?project=pizzabot-0.25&model=pizzabot-0.25_modifiers HTTP/1.1" 200 71 "-" "axios/0.18.0"
rasa_1           | 2019-06-27 17:56:02+0000 [-] "172.18.0.6" - - [27/Jun/2019:17:56:02 +0000] "DELETE /models?project=pizzabot-0.25&model=pizzabot-0.25_modifiers HTTP/1.1" 200 25 "-" "axios/0.18.0"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:17:56:02 +0000] "POST /api/agent/1/train HTTP/1.1" 200 811 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:33 +0000] "POST /api/agent/1/category/1/saying HTTP/1.1" 200 134 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:34 +0000] "GET /api/agent/1/saying?skip=0&limit=5&field=id&direction=DESC&loadCategoryId=true HTTP/1.1" 200 425 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:34 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:38 +0000] "POST /api/agent/1/category/1/saying HTTP/1.1" 200 141 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:39 +0000] "GET /api/agent/1/saying?skip=0&limit=5&field=id&direction=DESC&loadCategoryId=true HTTP/1.1" 200 420 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:39 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:45 +0000] "POST /api/agent/1/category/1/saying HTTP/1.1" 200 130 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:46 +0000] "GET /api/agent/1/saying?skip=0&limit=5&field=id&direction=DESC&loadCategoryId=true HTTP/1.1" 200 427 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:46 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:54 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/action/create?filter=&page=1&pageSize=5&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:54 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/action/create?filter=&page=1&pageSize=5&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:54 +0000] "GET /api/agent/1/action?field=actionName&skip=0&limit=-1 HTTP/1.1" 200 1435 "http://localhost:8080/agent/1/action/create?filter=&page=1&pageSize=5&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:54 +0000] "GET /api/agent/1/action?field=actionName&skip=0&limit=-1 HTTP/1.1" 200 1435 "http://localhost:8080/agent/1/action/create?filter=&page=1&pageSize=5&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:20:54 +0000] "GET /api/agent/1/action?field=actionName&skip=0&limit=-1 HTTP/1.1" 200 1435 "http://localhost:8080/agent/1/action/create?filter=&page=1&pageSize=5&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:21:04 +0000] "GET /api/agent/1/action?field=actionName&skip=0&limit=-1 HTTP/1.1" 200 1435 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:21:04 +0000] "GET /api/agent/1/action?field=actionName&skip=0&limit=-1 HTTP/1.1" 200 1435 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:21:04 +0000] "GET /api/agent/1/category?skip=0&limit=-1 HTTP/1.1" 200 511 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:21:04 +0000] "GET /api/agent/1/saying?skip=75&limit=5&field=id&direction=DESC&loadCategoryId=true HTTP/1.1" 200 547 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:21:04 +0000] "GET /api/agent/1/action?field=actionName&skip=0&limit=-1 HTTP/1.1" 200 1435 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:21:05 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:22:11 +0000] "GET /api/agent/1/saying?skip=70&limit=5&field=id&direction=DESC&loadCategoryId=true HTTP/1.1" 200 631 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:22:11 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:22:13 +0000] "GET /api/agent/1/saying?skip=65&limit=5&field=id&direction=DESC&loadCategoryId=true HTTP/1.1" 200 348 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
nginx_1          | 172.18.0.1 - - [27/Jun/2019:18:22:13 +0000] "GET /api/agent/1/keyword?skip=0&limit=-1 HTTP/1.1" 200 775 "http://localhost:8080/agent/1/dialogue?filter=&page=16&tab=sayings" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0" "-"
```

0.25 release
Firefox

**Comment by RussellSPG** [comment]
Update: I believe this issue is happening when going from different windows on browser/PC, just occurred again. Also, the page numbers disappear.

![image](https://user-images.githubusercontent.com/42249324/60293451-41325700-98ed-11e9-91c8-071aff4066a1.png)

And I have about 17 pages or so.

**Comment by RussellSPG** [comment]
Another update to this issue, this is also happening when editing an action, then hitting back to go to the sayings screen again.

**Comment by RussellSPG** [comment]
Just an update to this issue: When I hit the refresh button on the browser, the pages go back to the latest. For example, if I hit the back button and go all the way back to the first page and it glitches out, and I hit refresh, it will take me all the way back to the latest page.

**Comment by wrathagom** [comment]
Related to #backrelated to #799

**Comment by RussellSPG** [comment]
Resolved for 27.2 though I did get again after refreshing the browser. No errors, and couldn't reproduce as of now, but definitely solid. Tested the different functionalities too, pagination problem didn't appear.

**Comment by dcalvom** [comment]
We need to confirm this has been fixed

**Comment by dakavisa** [comment]
Closed by 3adc7cbd57433d4a79e16569249406e39543ba3b
